### ISSUE TITLE
Suggestion: Simlify left menu

### ISSUE THREAD

**Issue by brawaru** [mastodon/mastodon#12334]
### Pitch

Currently left menu footer feels “flooded” with all these redundant items. Good UI is mostly a simple one, so consider simplifying it by hiding useless elements.

I've made a plan how in my opinion left menu can be simplified.

---

#### Compose panel

<p align="center">
<img alt="Example of compose panel" src="https://user-images.githubusercontent.com/10401817/68530639-a89cf080-033c-11ea-88e4-98132334593b.png" />
</p>

**Changes**:

- “Muted users”, “Muted words”, “Blocked users”, “Hidden domains” are now hidden under single entity “Manage filtering”, which should redirect to special tab where you can manage all of these
  - **BLOCKED-ON**: such tab needs to be created

- “Follow requests” is not displayed unless your profile is marked private

**Removals**:

- “Favorites”, “Lists”, “Preferences” are useless to show as they duplicate main right side menu

- “Edit profile” link under the username is removed because it is duplicating menu

**Additions**:

- On top of username added bold “Tooting as:”, it may be instead replaced with usernames or superseded by “`{followers, plural, one {# follower} other {# followers}}`” or “`{posts, plural, one {# toot} other {# toots}}`” under the name

**Considerations**:

- Use **⋮** vertical ellipsis instead of **⌄** down arrow to avoid misleading with understanding that it will open menu and will not show other accounts

---

#### Footer

<p align="center">
<img alt="Example of footer" src="https://user-images.githubusercontent.com/10401817/68530218-062f3e00-0339-11ea-834c-2e7a3b063569.png" />
</p>

**Changes**:

- “Hotkeys” may better be renamed to “Keyboard navigation”: it may help with understanding the meaning and finding it

- “About this server”, “Developers”, “Documentation” can be moved under dropdown menu “More” as they don't present so much importance to stay always vilible

- Higher line height makes it less likely to misclick any item and more friendly to “touch” pressing

**Removals**:

- “Log out” is already present in the user menu, it makes no sense to duplicate it

- Same applies to “Account settings”: it is already present in Settings

- Dot separators present as normal text

<details>
<summary>Some of CSS changes</summary>

```diff
  .getting-started__footer ul li {
+   /* Avoid breaking in the middle of words */
+   word-break: break-all;
+   white-space: nowrap;
+   /* Margin not working nicely here, but padding is */
+   padding-right: 15px;
  }

  .getting-started__footer ul {
+  line-height: 30px;
  }

+ /* Emulate down arrowhead */
+ .footer_more span::after {
+   content: "⌄";
+   position: relative;
+   top: -3px;
+   left: 3px;
+   font-size: 15px;
+   font-weight: bold;
+ }
```
</details>

### Motivation

This may or may not improve user experience by introducing only single straight paths to do things and not getting caught by tons of entities and different ways to accept same pages.

### Related Issues

- #12814: Display open source notice only when hovering (?)

**Comment by brendanjones** [comment]
I agree with the removals but honestly I think go a step further and remove the entire thing, it's obsolete: https://github.com/mastodon/mastodon/issues/23355
